helm/ffc-sfd-file-processor-infra/values.yaml

```diff
@@ -1,3 +1,15 @@
 # For documentation on how to use, please go here: https://github.com/DEFRA/adp-aso-helm-library?tab=readme-ov-file#adp-platform-azure-service-operator-helm-library-chart.
 # You may create Queues, Topics, Subscriptions, Storage Accounts, Databases and Role Assignments on these here.
+storageAccounts:
+  - name: sfd
+    roleAssignments:
+      - roleName: 'DataWriter'
+      - roleName: 'DataReader'
+    blobContainers:  
+      - name: ffc-sfd-file-processor
 
```

Comment:
For information only but this is what was missing that I added yesterday, we had not given this application the necessary permissions to use Service Bus so app was resulting in permission denied errors. 

Resolved by reviewing documentation. Every application that interacts with service bus must have permission granted even though we created the infrastructure on the receiver service.